src/acceptance/app/recurring_schedule_policy_test.go

```diff
@@ -0,0 +1,156 @@
+package app
+
+import (
+	"acceptance/config"
+	"fmt"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+	"io/ioutil"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var _ = Describe("AutoScaler recurring schedule policy", func() {
+	var (
+		appName              string
+		appGUID              string
+		instanceName         string
+		dayOfMonthOrWeek     string
+		location             *time.Location
+		startTime            time.Time
+		endTime              time.Time
+		policyByte           []byte
+	)
+
+	BeforeEach(func() {
+		instanceName = generator.PrefixedRandomName("autoscaler", "service")
+		createService := cf.Cf("create-service", cfg.ServiceName, cfg.ServicePlan, instanceName).Wait(cfg.DefaultTimeoutDuration())
+		Expect(createService).To(Exit(0), "failed creating service")
+
+		initialInstanceCount := 1
+		appName = generator.PrefixedRandomName("autoscaler", "nodeapp")
+		countStr := strconv.Itoa(initialInstanceCount)
+		createApp := cf.Cf("push", appName, "--no-start", "-i", countStr, "-b", cfg.NodejsBuildpackName, "-m", cfg.NodeMemoryLimit, "-p", config.NODE_APP, "-d", cfg.AppsDomain).Wait(cfg.DefaultTimeoutDuration())
+		Expect(createApp).To(Exit(0), "failed creating app")
+
+		guid := cf.Cf("app", appName, "--guid").Wait(cfg.DefaultTimeout)
+		Expect(guid).To(Exit(0))
+		appGUID = strings.TrimSpace(string(guid.Out.Contents()))
+
+		Expect(cf.Cf("start", appName).Wait(cfg.DefaultTimeout * 2)).To(Exit(0))
+		waitForNInstancesRunning(appGUID, initialInstanceCount, cfg.DefaultTimeoutDuration())
+	})
+
+	AfterEach(func() {
+		Expect(cf.Cf("delete", appName, "-f", "-r").Wait(cfg.CfPushTimeoutDuration())).To(Exit(0))
+
+		deleteService := cf.Cf("delete-service", instanceName, "-f").Wait(cfg.DefaultTimeoutDuration())
+		Expect(deleteService).To(Exit(0))
+	})
+
+	Context("when scale out by recurring schedule", func() {
+
+		JustBeforeEach(func() {
+			timeZone := "GMT"
+			location, _ = time.LoadLocation(timeZone)
+			startTime, endTime = getStartAndEndTime(location)
+
+			policyStr := setRecurringScheduleDateTime(policyByte, timeZone, startTime, endTime, dayOfMonthOrWeek)
+			bindService := cf.Cf("bind-service", appName, instanceName, "-c", policyStr).Wait(cfg.DefaultTimeoutDuration())
```

Comment:
This doesn't make sense. We bind after we already started?
It works but requires a restart, which means more time.